all_repos/bitbucket_server_api.py

```diff
@@ -0,0 +1,32 @@
+import json
+import urllib.request
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+
+class Response(NamedTuple):
+    values: Any
+    next: Optional[int]
+    links: Any
```

Comment:
should this be `Optional[Dict[str, Any]]` ?

Reply:
done.